browser/src/control/jsdialog/Util.Dropdown.js

```diff
@@ -72,6 +72,12 @@ JSDialog.OpenDropdown = function (id, popupParent, entries, innerCallback, popup
 			break;
 		}
 
+		if (entries[i].type === 'iconview') {
```

Comment:
please  just use generic `json` type which will do the work, no need for a new type

`{ type: 'json', content: <here old JSON> }`